src/include/quick-lint-js/parse.h

```diff
@@ -906,8 +904,10 @@ class parser {
           break;
         case token_type::kw_if:
         case token_type::number:
+        case token_type::kw_debugger:
```

Comment:
Nit: Sort these cases alphabetically:
```suggestion
        case token_type::kw_debugger:
        case token_type::kw_if:
        case token_type::number:
```